ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -107,8 +101,15 @@ def _check_volumedriver(vdisk_name, storagedriver_guid, vdisk_size=VDISK_CHECK_S
         :return: True if succeeds
         :rtype: bool
         """
-
-        VDiskController.create_new(vdisk_name, vdisk_size, storagedriver_guid)
+        try:
+            VDiskController.create_new(vdisk_name, vdisk_size, storagedriver_guid)
+        except FileExistsException:
+            # can be ignored until fixed in framework
+            # https://github.com/openvstorage/framework/issues/1247
+            return True
+        except Exception as ex:
+            logger.failure("Creation of the vdisk failed. Got {0}".format(str(ex)))
```

Comment:
Its always better to use logger.exception, then the stacktrace is printed for you and you dont't have to do it manually